mmpose/datasets/datasets/top_down/topdown_jhmdb_dataset.py

```diff
@@ -185,8 +186,10 @@ def _load_coco_keypoint_annotation_kernel(self, img_id):
                 'joints_3d': joints_3d,
                 'joints_3d_visible': joints_3d_visible,
                 'dataset': self.dataset_name,
-                'bbox_score': 1
+                'bbox_score': 1,
+                'bbox_id': str(img_id) + str(bbox_id)
```

Comment:
take care here, since sorting in str and int is different.

use `f'{img_id}{bbox_id:03}'`